src/filters/noise/NoiseFilter.js

```diff
@@ -29,21 +29,37 @@ export default class NoiseFilter extends core.Filter
         );
 
         this.noise = 0.5;
```

Comment:
One very small thing that would be nice is to allow a single constructor argument for the noise value with a default for `0.5`. 

```js
// Current, non-default value
const filter = new PIXI.filters.NoiseFilter();
filter.noise = 0.1;
displayObject.filters = [ filter ];

// With constructor arg
displayObject.filters = [ new PIXI.filters.NoiseFilter(0.1) ];
```